resources/energyplus/ProposedEnergy+.idd

```diff
@@ -32556,9 +32887,9 @@ Coil:Heating:DX:VariableSpeed,
         \memo (includes electric compressor and outdoor fan), variable-speed, with defrost
         \memo controls. Requires two to ten sets of performance data and will interpolate between
         \memo speeds.
-   \extensible:7
-   \min-fields 25
-   \max-fields 88
+        \extensible:7
```

Comment:
This should be bumped to 9. I will do that on the Coils PR, but just a note that there could be other occurrences where bumping extensible was needed.